.gitignore

```
@@ -33,6 +33,7 @@ pip-wheel-metadata
 # Documentation
 */_build/*
 */site/*
+site/
```

Possibly you meant this:

```suggestion
/site
```

Isn't this covered by line 35?

---

I was surprised too, but no, line 35 does not ignore the root `site` directory :confused: